app/src/main/java/com/hapramp/views/post/PostItemView.java

```diff
@@ -222,6 +228,44 @@ private void bind(final Feed feed) {
     attachListenersOnStarView();
     attachListerOnAuthorHeader();
     attachListenerForOverlowIcon();
+    attachListenerOnPayoutValue();
+  }
+
+  private void setSteemEarnings(Feed feed) {
+    try {
+      StringBuilder stringBuilder = new StringBuilder();
+      DecimalFormat df = new DecimalFormat("#.###");
+      df.setRoundingMode(RoundingMode.CEILING);
+      double pendingPayoutValue = Double.parseDouble(feed.getPendingPayoutValue().split(" ")[0]);
+      double totalPayoutValue = Double.parseDouble(feed.getTotalPayoutValue().split(" ")[0]);
+      double curatorPayoutValue = Double.parseDouble(feed.getCuratorPayoutValue().split(" ")[0]);
+      if (pendingPayoutValue > 0) {
+        //payout is pending
+        String cashoutTime = MomentsUtils.getFormattedTime(feed.getCashOutTime());
+        stringBuilder
+          .append("Pending Payout")
+          .append("\n")
+          .append("$")
+          .append(df.format(pendingPayoutValue))
+          .append("\n")
+          .append(cashoutTime);
+        briefPayoutValueString = "$" + df.format(pendingPayoutValue);
+      } else {
+        //cashed out
+        stringBuilder
+          .append("Past Payout $")
+          .append(String.valueOf(df.format(totalPayoutValue + curatorPayoutValue)))
+          .append("\n")
+          .append("- Author $").append(df.format(totalPayoutValue))
+          .append("\n")
+          .append("- Curator $").append(df.format(curatorPayoutValue));
+        briefPayoutValueString = "$" + (df.format(totalPayoutValue + curatorPayoutValue));
+      }
+      detailedPayoutValueString = stringBuilder.toString();
```

Comment:
Where is the `detailedPayoutValueString` rendered?

Reply:
This was for the detailed payout view. This will show up when the user taps on the payout view.
But now we are not allowing the user to view it.

Reply:
Please remove it if it's not used anymore.